### ISSUE TITLE
How to style form input element with different id?

### ISSUE THREAD

**Issue by CodeCurosity** [themeum/kirki#1143]
body #form_wrapper_1  input[type=text] {border: 1px solid red;}

body #form_wrapper_2 input[type=text] {border: 1px solid orange;}

Is there an easy way to have different styles for same input with different id's in one configuration?

**Comment by aristath** [comment]
what control are you using?

**Comment by CodeCurosity** [comment]
quite a few actually, i am using color, slider, typography e.t.c i just gave an example of one input.

**Comment by aristath** [comment]
Perhaps I misunderstood....
You're talking about styling the actual customizer controls? Or automatically generating styles for your front-end?

**Comment by CodeCurosity** [comment]
automatically generating styles for frontend form.

**Comment by CodeCurosity** [comment]
@aristath anything?

**Comment by aristath** [comment]
Have you read the docs for the output argument? It's all there... https://aristath.github.io/kirki/docs/arguments/output.html

**Comment by CodeCurosity** [comment]
Yes i have already implemented everything but that is for single instance.

For example i have 2 forms in one page and each form contains one text input field, both forms have unique id's.

I have a text input in customizer where you define id for the form you want to style, it all works just fine for one form but when you change the id it applies all those styles to new form and all the styling for previous form is gone.

In short i am looking for a way to load new configuration when the form id is changed but still keep the styling saved for previous form so they can switch to another form and change styling, makes sense?

**Comment by aristath** [comment]
From the docs: https://aristath.github.io/kirki/docs/arguments/output.html#arguments-you-can-use-inside-each-output-array

The output argument is defined as an array of arrays, so you can add multiple styles like this:

``` php
    'output' => array(
        array(
            'element'  => '.variable-top-border',
            'property' => 'border-top-width',
            'units'    => 'px',
        ),
        array(
            'element'  => '.variable-bottom-border',
            'property' => 'border-bottom-width',
            'units'    => 'px',
        ),
    ),
```

Furthermore, the `element` can be an array, so if you want a style to be applied to multiple IDs then you can do it like this:

``` php
    'output' => array(
        array(
            'element'  => array(
                '#id-1',
                '#id-2',
                '#id-3',
            ),
            'property' => 'border-top-width',
            'units'    => 'px',
        ),
    ),
```

But what you're describing is a bit weird... What you could do is add a repeater control that will contain your ID textfield and your other styling fields, so users can add as many custom styles as they want.
Un that case however, you'll have to completely custom-code the CSS output as it won't work by default.

**Comment by CodeCurosity** [comment]
Yes the repeater field was ideal but i have plenty of fields and i had to group them in panels and sections so that is a no go, i guess i will just create multiple instances of all fields and group them in ID's, thanks anyways great product.
